### ISSUE TITLE
Unable to resolve AccessibilityInfo

### ISSUE THREAD

**Issue by kwizeradev** [halilb/rn-credit-card#24]
Unable to resolve "AccessibilityInfo" from "node_modules\rn-credit-card\node_modules\react-native\Libraries\react-native\react-native-implementation.js"
